src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -36,7 +36,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         public CSharpParseOptions(
-            LanguageVersion languageVersion = LanguageVersion.CSharp6,
+            LanguageVersion languageVersion = LanguageVersion.CSharp7,
```

Comment:
Are we switching the default language to be v7?
Just wondering. Introducing v7 does not necessary means we would have it as default right away.